### PR TITLE
Proper library for fmod 

### PR DESCRIPTION
**Pull request by sumesh13**

### CODE REVIEW

File: tensorflow/compiler/xla/service/gpu/elemental_ir_emitter.cc [hunk 1]
```diff
@@ -178,7 +178,7 @@ StatusOr<llvm::Value*> GpuElementalIrEmitter::EmitFloatBinaryOp(
   switch (op->opcode()) {
     case HloOpcode::kRemainder: {
 
-      return EmitLibdeviceMathCall("fmod", {lhs_value, rhs_value},
+      return EmitLibdeviceMathCall("__ocml_fmod", {lhs_value, rhs_value},
```
Comment (deven-amd) [inline tensorflow/compiler/xla/service/gpu/elemental_ir_emitter.cc]: will `__ocml_fmod` work for the CUDA path too? or will this change break the CUDA path?
same question applies for the similar change in PR 395.

Comment (sumesh13) [reply tensorflow/compiler/xla/service/gpu/elemental_ir_emitter.cc]: It wont work for CUDA path. Our XLA path is currently hardcoded in many places for rocm.  I have two branches - starting with branch device-functions- that are for making both CUDA and rocm paths co-exist. As soon as PR 396 is merged will test the first one.

**Comment by sumesh13** [comment]
retest this please

**Comment by whchung** [comment]
jenkins: retest this please

**Comment by whchung** [comment]
@sumesh13 you'll need to prefix "jenkins:" before the command gets parsed

**Comment by sumesh13** [comment]
rocm-v2 fail should be unrelated

**Pull request merged**